activate.sh

```diff
@@ -28,7 +28,7 @@ autoenv_init() {
 ${_file}"
 				fi
 			fi
-			command -v chdir >/dev/null 2>&1 && chdir .. || builtin cd ..
+			command -v chdir >/dev/null 2>&1 && chdir (pwd -P)/.. || builtin cd (pwd -P)/..
```

Comment:
Not sure about this, it might fix an issue we haven't found yet (initite looping), but it might also create new ones. I'll test this.